pandas/_libs/period.pyx

```diff
@@ -33,10 +33,11 @@ from pandas._libs import tslib
 from pandas._libs.tslib import Timestamp, iNaT, NaT
 from tslibs.timezones cimport (
     is_utc, is_tzlocal, get_utcoffset, get_dst_info, maybe_get_tz)
-from tslib cimport _nat_scalar_rules
 
 from tslibs.parsing import parse_time_string, NAT_SENTINEL
 from tslibs.frequencies cimport get_freq_code
+from tslibs.nattype import nat_strings
+from tslibs.nattype cimport _nat_scalar_rules
```

Comment:
in followup I could de-privatize this ``_nat_scalar_rules``)

Reply:
That works.  I'm also ok with getting rid of it, since  `op != Py_NE` is pretty easy.